cmd/truchaind/migrate.go

```diff
@@ -0,0 +1,128 @@
+package main
+
+import (
+	"fmt"
+	"io/ioutil"
+	"sort"
+	"time"
+
+	"github.com/TruStory/truchain/cmd/truchaind/migration/v0_3"
+	"github.com/pkg/errors"
+	"github.com/spf13/cobra"
+	"github.com/tendermint/tendermint/types"
+
+	"github.com/cosmos/cosmos-sdk/codec"
+	"github.com/cosmos/cosmos-sdk/server"
+	sdk "github.com/cosmos/cosmos-sdk/types"
+	"github.com/cosmos/cosmos-sdk/version"
+	extypes "github.com/cosmos/cosmos-sdk/x/genutil"
+)
+
+const (
+	flagGenesisTime     = "genesis-time"
+	flagChainID         = "chain-id"
+	flagResetValidators = "reset-validators"
+)
+
+var migrationMap = extypes.MigrationMap{
+	"v0.3": v0_3.Migrate,
```

Comment:
This is the migrate to, not migrate from version right? In that case, shouldn't it be v0.3.1 since v0.3 already exists?